src/PersonReader.java

```diff
@@ -1,6 +1,66 @@
-public class PersonReader {
-    public static void main(String[] args) {
-        // Write your code here ... ⬇
 
+    import java.awt.*;
+    import java.util.Scanner;
+    import javax.swing.JFileChooser;
+    import java.io.FileNotFoundException;
+    import java.io.File;
+    import java.util.StringTokenizer;
+
+
+    public class PersonReader
+{
+
+
+
+    public static void main(String[] args)
+    {
+
+        boolean hasFile = false;
+        File selectedFile = new File("");
+        JFileChooser choose = new JFileChooser();
+        String filename = "classmates.csv";
+        String lines = "";
+        String currToken = "";
+
+
+
+        choose.setCurrentDirectory(new File("C:\\Users\\winso\\OneDrive\\Documents\\UC"));
```

Comment:
so this line wouldn't work on any computer other than yours. 

This however will work wherever your run the code
```suggestion
        choose.setCurrentDirectory(new File(System.getProperty("user.dir")););
```